### DISCUSSION TITLE
!  işareti sorunu

### DISCUSSION THREAD

**Discussion by Metheus5809** [IBAX-io/go-ibax#1063] [Soru Cevap - Türkçe] [answered]
Cüzdan da profil avatarının yanında "  !  " işareti çıkıyor ve bu yazıyor, çözümü nedir önemli bir şey mi?

Failed to establish connection to the WebSocket server. Check your configuration

WebSocket sunucusuyla bağlantı kurulamadı. Yapılandırmanızı kontrol edin.

![Screenshot_20220127-123715_Chrome](https://user-images.githubusercontent.com/74926294/151332356-c48c944c-8392-4b86-ad27-2b2731a03331.jpg)

**Comment by Yusufknk** [comment]
Bu sorunu weaver wallet lokasyonu altında paylaşın.

**Comment by hes91** [comment]
Bu bir sorun olarak gözükmüyor.Çünkü anlık birşey sadece ilk sayfaya girdiğinizde ünlem işareti çıkıp eski haline dönüyor.Kalıcı birşey kalıcı bir sorun değil.

**Comment by ibaxio** [comment, marked as answer]
Ağ sorunu, lütfen tekrar deneyin.
